torch/csrc/api/include/torch/nn/modules/activation.h

```diff
@@ -120,5 +120,31 @@ class TORCH_API LogSigmoidImpl : public torch::nn::Cloneable<LogSigmoidImpl> {
 
 TORCH_MODULE(LogSigmoid);
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ PReLU ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/// Applies the PReLU function element-wise.
+/// See https://pytorch.org/docs/master/nn.html#torch.nn.PReLU to learn
+/// about the exact behavior of this module.
+class TORCH_API PReLUImpl : public torch::nn::Cloneable<PReLUImpl> {
+ public:
+  PReLUImpl() : PReLUImpl(PReLUOptions()) {}
+  explicit PReLUImpl(const PReLUOptions& options_);
```

Comment:
Would we be able to combine the above two constructors and just have the following?
```cpp
  explicit PReLUImpl(const PReLUOptions& options_ = {});
```